eslzArm/README-AzureChina.md

```diff
@@ -27,7 +28,7 @@ New-AzManagementGroupDeployment -Name $DeploymentName `
                                 -Verbose
 
 # Deploy core policy definitions to ESLZ intermediate root management group
-# Note: If you get an error message when deploying any policy set definition because policy definitions could not be found, please retry the PowerShell command.
+# Note: If your ProvisioningState is "Failed", please go to your top level management group prefix under Management Groups in the Azure Portal, under Governance click "Deployments", click on the deployment with the post-fix "-policy1". If the Operation Details show that the policy set definition request is invalid because some policy definition could not be found, this is often due to a race condition which happens during which ARM is deploying both the policy set definitions and the policy definitions at the same time. Please wait for awhile and re-run the Azure Powershell command to deploy the mcPolicies.json template and this would result in a "Succeeded" ProvisioningState.
```

Comment:
Can we rather say this is due to a known replication delay, as the dependency graph is indeed respected, but due to how traffic is being routed the replication may not be completed when trying to GET the definitions that just got created?

Reply:
As documented @ https://github.com/Azure/Enterprise-Scale/blob/main/docs/EnterpriseScale-Known-Issues.md#deploying-the-reference-implementation-fails-due-to-policy--cannot-be-found-404

Reply:
@krnese yeah that makes sense, I have reworded the explanation and committed it below